src/components/ethContracts.js

```diff
@@ -228,7 +216,14 @@ class EthContracts extends React.Component {
             <Table.Column title='App' dataIndex='app' key='app'/>
             <Table.Column title='Source' dataIndex='website' key='website'
               render={url => (
-                <a href={url} target={"_blank"}>Link</a>
+                <a  className="lattice-a" 
+                    href={url} 
+                    target={"_blank"} 
+                    rel={"noopener noreferrer"}
+                    key={`a-${url}`}
```

Comment:
I couldn't figure out where these contract urls (e.g., `v2_curve`, `v2_aave`) get loaded (explain plz); however, some of them need to be updated:
 - Balancer: https://docs.balancer.fi/v/v1/smart-contracts/addresses
 - Curve: https://curve.readthedocs.io/ref-addresses.html
 - Gnosis Safe: should update pack for v1.3.0? (https://github.com/gnosis/safe-contracts/blob/v1.3.0/CHANGELOG.md)
 - Opyn v2: link out of date, but no better link found...
 - SushiSwap: Oooops — **I lol'd when I found the ADDRESSES & LINK forwards to UniSwap 😂**
 - UniSwap: https://github.com/Uniswap/v3-periphery/blob/main/deploys.md

Reply:
They are loaded from a JSON file stored in a public S3 bucket. Will update it.

Reply:
2e23175